oam/install_scripts/mcs-storagemanager.service.in

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=storagemanager
+PartOf=mcs-workernode.service
+Before=mcs-workernode.service
+ConditionPathExists=/etc/columnstore/storagemanager.cnf
+# FailureAction="exit"
+# FailureActionExitStatus=0
```

Comment:
If we can figure this out we can probably have unit have a success status w/o being active. Right now we have failure status and inactive which does not satisfy myself.